server/Cargo.lock

```diff
@@ -1829,7 +1829,7 @@ dependencies = [
  "httpdate",
  "itoa",
  "pin-project-lite",
- "socket2 0.5.6",
+ "socket2 0.4.10",
```

Comment:
This must be some Cargo bug, I've seen it downgrade socket2 multiple times already, but whenever I revert it compiles too. Let's keep the new version.

```suggestion
 "socket2 0.5.6",
```